src/twisted/test/test_twistd.py

```diff
@@ -1911,5 +1956,83 @@ def test_hooksNotCalled(self):
 
 
 
+class ExitWithSignalTests(unittest.TestCase):
+
```

Comment:
Should twistedchecker flag this?  It didn't for me (though I did miss another "too many blank lines" warning later in the class.